aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -73,11 +74,17 @@
 import software.amazon.awssdk.services.glue.model.Table;
 import software.amazon.awssdk.services.glue.model.TableInput;
 import software.amazon.awssdk.services.glue.model.UpdateDatabaseRequest;
+import software.amazon.awssdk.services.s3.model.Tag;
 
 public class GlueCatalog extends BaseMetastoreCatalog
     implements Closeable, SupportsNamespaces, Configurable<Configuration> {
 
   private static final Logger LOG = LoggerFactory.getLogger(GlueCatalog.class);
+  private static final String S3_FILE_IO_IMPL = "org.apache.iceberg.aws.s3.S3FileIO";
+  private static final String S3_TAGS_FORMAT = "s3.tags.format";
+  private static final String S3_TAGS_NAMESPACE = "s3.tags.namespace";
+  private static final String S3_TAGS_TABLE = "s3.tags.table";
```

Comment:
also adding tags for table and namespace names might not be ideal, because people can rename table and these information could change. With the generic tagging solution described above, we can in the future add tags using some stable Iceberg table properties, such as the Iceberg table UUID.

Reply:
Agreed, I have made the changes.